models/layerzero/arbitrum/layerzero_arbitrum_send.sql

```diff
@@ -0,0 +1,218 @@
+{{ config(
+    schema = 'layerzero_arbitrum',
+    alias = 'send',
+    partition_by = ['block_date'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['block_time', 'tx_hash', 'user_address', 'trace_address', 'source_chain_id', 'destination_chain_id', 'currency_contract'],
+    )
+}}
+
+{% set transaction_start_date = "2022-03-15" %}
+{% set endpoint_contract = "0x66a71dcef29a0ffbdbe3c6a460a3b5bc225cd675" %}
+{% set native_token_contract = "0x82af49447d8a07e3bd95bd0d56f35241523fbab1" %}
+
+WITH chain_list AS (
+    SELECT chain_id,
+        chain_name,
+        endpoint_address
+    FROM {{ ref('layerzero_chains') }}
+),
+
+send_detail AS (
+    SELECT CAST(110 AS integer) AS source_chain_id,
+        s.call_tx_hash as tx_hash,
+        s.call_block_number as block_number,
+        s._dstChainId AS destination_chain_id,
+        s.contract_address,
+        s.call_block_time AS block_time,
+        s.call_trace_address AS trace_address,
+        s._adapterParams AS adapter_params,
+        s._refundAddress AS refund_address,
+        s._zroPaymentAddress AS zro_payment_address,
+        t.`from` AS user_address,
+        t.`to` AS transaction_contract_address,
```

Comment:
Since we have table alias `t` here, we can remove ` for better compatibility.
```suggestion
        t.from AS user_address,
        t.to AS transaction_contract_address,
```